packages/helper-insert-link/index.js

```diff
@@ -182,33 +145,35 @@ function replace(portion, match, captureGroup) {
   };
 }
 
-export default function(el, regex, mapping, captureGroup = '$1') {
-  if (!(el instanceof HTMLElement)) {
-    throw new Error('must be called with a DOM element');
+export default function(blob, regex, plugin, meta = {}) {
```

Comment:
`meta` can be used to pass additional informations down to the resolver function. This is only used in the `bower` and `npm` manifest plugin to distinguish between different resolver behaviour see https://github.com/stefanbuck/OctoLinker/blob/850801fd5e2cfd91db428e3f0173956c03c756aa/packages/plugin-npm-manifest/index.js#L13-L25